src/drivers/dshot/DShot.cpp

```diff
@@ -668,25 +470,31 @@ void DShotOutput::retrieveAndPrintESCInfoThreadSafe(int motor_index)
 	DShotTelemetry::decodeAndPrintEscInfoPacket(output_buffer);
 }
 
-int DShotOutput::requestESCInfo()
+int
+DShot::request_esc_info()
```

Comment:
I've been doing return type on the same line as the function name (https://google.github.io/styleguide/cppguide.html#Function_Declarations_and_Definitions). 

Do you have a strong preference?

Reply:
I can make them all on the same line, in current master they are a mix of both styles.

Reply:
Done.